File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.go
func twoSum(numbers []int, target int) []int {
    i, j := 0 , len(numbers) - 1;

    for i <= j {
        a := numbers[i];
        b := numbers[j];

        if a+b > target {
            j--;
        } else if a+b < target {
            i++;
        } else {
            return []int{i+1 , j+1};
        }
    }

    return []int{};
    
}

File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        n = len(nums)
        i = 0
        j = n-1

        while i < j:
            s = nums[i] + nums[j]

            if s > target:
                j-=1
            elif s < target:
                i+=1
            else:
                return [i+1, j+1]
        
        